docker/ssl/nginx-image/start.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+sleep 3
+
+echo build starting nginx config
+
+echo replacing ___my.example.com___/$MY_DOMAIN_NAME
+echo replacing ___LETSENCRYPT_IP___/$LETSENCRYPT_PORT_80_TCP_ADDR
```

Comment:
By using $LETSENCRYPT_* variables you are using a deprecated docker compose feature https://docs.docker.com/compose/link-env-deprecated/ . This got me baffled when I first read the script, wondering where these were coming from (the [PR](https://github.com/xibosignage/xibo-docker/pull/47/files) from which you ported the behaviour is more than an year old indeed, which is a *lot* of time in the Docker world :stuck_out_tongue: )
I would suggest to use once again the Docker embedded DNS to resolve to obtain the address of the letsencrypt container.